plugins/view-transitions/includes/theme.php

```diff
@@ -88,6 +89,184 @@ function plvt_sanitize_view_transitions_theme_support(): void {
 	$_wp_theme_features['view-transitions'] = $args;
 }
 
+/**
+ * Registers the default view transition animations and fires an action to register additional ones.
+ *
+ * @since 1.0.0
+ * @access private
+ *
+ * @param PLVT_View_Transition_Animation_Registry $animation_registry Registry instance to register animations on.
+ */
+function plvt_register_view_transition_animations( PLVT_View_Transition_Animation_Registry $animation_registry ): void {
+	// Register default animations.
+	$animation_registry->register_animation(
+		'fade', // This is how view transitions are animated without any extra CSS.
+		array(
+			'use_stylesheet'              => false,
+			'use_global_transition_names' => true,
+			'use_post_transition_names'   => true,
+		)
+	);
```

Comment:
In future are we going to add fade animations?

Reply:
How do you mean that? The fade animation is what happens by default, if you don't configure view transitions in any more specific way.

Reply:
Are we planning to add animation effects like `top`, `bottom`, etc., in the future, similar to those listed on W3Schools - https://www.w3schools.com/w3css/w3css_animate.asp?